documentation/src/main/asciidoc/topics/server_protocols_rest.adoc

```diff
@@ -67,8 +67,9 @@ The <cors-rule> element can be configured as follows:
 | allow-credentials | Enable CORS requests to use credentials | no
 | allowed-origins | A comma separated list used to set the CORS 'Access-Control-Allow-Origin' header to indicate the response can be shared with the origins | yes
 | allowed-methods | A comma separated list used to set the CORS 'Access-Control-Allow-Methods' header in the preflight response to specify the methods allowed for the configured origin(s) | yes
-| max-age-seconds | The amount of time CORS preflight request headers can be cached | no
+| allowed-headers | A comma separated list used to set the CORS 'Access-Control-Allow-Headers' in the preflight response to specify which headers can be used by the configured origin(s) | no
```

Comment:
@ryanemerson there's actually a separate PR for REST docs where I can standardize these descriptions. I'll just comment on the line you added for now. the wording might be inconsistent with the other descriptions in this table but I'd suggest:

A comma separated list of headers that configured origins can use. The value sets the CORS 'Access-Control-Allow-Headers' response for preflight requests.

Maybe we could switch the order of the sentences but it sounds like there are two separate things to break out.

A comma separated list of headers that sets the CORS 'Access-Control-Allow-Headers' response header for preflight requests. The value specifies the headers that configured origins can use.

Reply:
@oraNod I'm happy for you to just add this line in your PR with the wording you prefer. I just copy and pasted this from our `.xsd` files ... you might want to update those as well for consistency.

Reply:
@ryanemerson sounds like a plan